Dockerfile.dev

```diff
@@ -1,19 +1,31 @@
-FROM ood:latest
+FROM rockylinux/rockylinux:8
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
 
 ARG USER=ood
 ARG UID=1000
 ARG GID=1000
 
-RUN dnf reinstall -y httpd-tools && \
-    dnf install -y \
-        strace lua python3 && \
-    dnf clean all && rm -rf /var/cache/dnf/*
+RUN dnf install -y https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm
+
+RUN cp /etc/yum.repos.d/ondemand-web.repo /etc/yum.repos.d/ondemand-nightly-web.repo && \
+    sed -i 's/latest/nightly/g' /etc/yum.repos.d/ondemand-nightly-web.repo && \
+    sed -i 's/ondemand-web/ondemand-web-nightly/g' /etc/yum.repos.d/ondemand-nightly-web.repo
```

Comment:
I don't know if really matters but you can condense this a little:

```
sed /etc/yum.repos.d/ondemand-web.repo -e 's/latest/nightly/g' \
  -e 's/ondemand-web/ondemand-web-nightly/g' > /etc/yum.repos.d/ondemand-nightly-web.repo
```

Reply:
```suggestion
RUN cp /etc/yum.repos.d/ondemand-web.repo /etc/yum.repos.d/ondemand-nightly-web.repo && \
        sed /etc/yum.repos.d/ondemand-web.repo -e 's/latest/nightly/g' \
          -e 's/ondemand-web/ondemand-web-nightly/g' > /etc/yum.repos.d/ondemand-nightly-web.repo
```